src/main/java/uk/sky/cqlmigrate/CassandraLockConfig.java

```diff
@@ -15,8 +15,8 @@ private CassandraLockConfig(Duration pollingInterval, Duration timeout, String c
     }
 
     @Override
-    public LockingMechanism getLockingMechanism(Session session, String keySpace) {
-        return new CassandraLockingMechanism(session, keySpace, consistencyLevel);
+    public LockingMechanism getLockingMechanism(Session session, String keySpace, String lockKeyspace) {
```

Comment:
As the `lockKeyspace` is only relevant for the `CassandraLockingMechanism` I think rather then it being a parameter of this method, it'd be better suited to being a field of the `CassandraLockConfig` class and being set and defaulted by the `CassandraLockConfigBuilder`. 

It'd save having to change the interface of this method and reduce the amount of files having to change to support this.